gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenFragment.java

```diff
@@ -225,6 +225,15 @@ private void showProjectSelector() {
     ProjectSelectorDialogFragment.show(getFragmentManager());
   }
 
+  private void showBasemapSelector() {
+    viewModel.showBasemapSelector();
+  }
+
+  private void showOfflineAreaManager() {
```

Comment:
"Manager" is used in our codebase to mean low level wrappers like `SettingsManager`. Maybe "showOfflineAreasUi" or similar? Then other classes woudl be "offlineAreasX" (where X is Fragment, etc.)

Reply:
Renamed OfflineAreas